src/server/native_controllers/native_controller_start.test.js

```diff
@@ -0,0 +1,75 @@
+let isBabelRegistered;
+jest.mock('../../babel-register', () => {
+  isBabelRegistered = true;
+});
+jest.mock('./native_controller_impl.js',
+  () => jest.fn(),
+  {
+    virtual: true
+  });
+
+let nativeController;
+let originalProcessArgv;
+beforeEach(() => {
+  isBabelRegistered = false;
+  jest.resetModules();
+
+  // eslint-disable-next-line
+  nativeController = require('./native_controller_impl.js'); // this is virtual, it doesn't really exist
+
+  process.send = jest.fn();
+  originalProcessArgv = process.argv;
+});
+
+afterEach(() => {
+  delete process.send;
+  process.argv = originalProcessArgv;
+  process.removeAllListeners();
```

Comment:
I think this will have unexpected consequences, perhaps instead we can use `jest.spyOn(process, 'on')` to stub `process.on()` instead?